modules/ROOT/pages/fd-publish-app-as-template.adoc

```diff
@@ -33,9 +33,9 @@ Templates do not include the authentication credentials that are specified for t
 
 == Procedure
 
-. Click the *Publish* button, which is in the top-right corner of the Flow Designer screen.
+. Select *Publish Template* under the down arrow which is in the top-right corner of the Flow Designer screen.
```

Comment:
Rather than "under the down arrow", I'd say, "in the drop-down menu which is ...". "Under" implies that "Publish Template" is visible without any action on the user's part.

Same comment for "under the down arrow" elsewhere.

Reply:
Fixed.